riscv/dts.cc

```diff
@@ -178,6 +181,88 @@ std::string dts_compile(const std::string& dts)
   return dtb.str();
 }
 
+std::string dtb_to_dts(const std::string& dtb)
```

Comment:
Can this be DRY'd with `dts_to_dtb`? Perhaps both should just call a new `exec_dtc` with different flags.

Reply:
Done.